mmv1/third_party/terraform/services/compute/resource_compute_instance_test.go.erb

```diff
@@ -6850,6 +6889,44 @@ resource "google_compute_instance" "foobar" {
 `, instance, enableConfidentialCompute)
 }
 
+<% unless version = 'ga' -%>
+func testAccComputeInstanceConfidentialHyperDiskBootDisk(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+}
+data "google_compute_image" "my_image" {
+  family    = "ubuntu-2004-lts"
```

Comment:
We should prefer the latest debian LTS (12, I think). If this has to be Ubuntu for reasons specific to the feature, we should prefer a `2204` image. Finally, if that's not available / we need specifically this version, ignore this comment!